client/src/common/components/Confirmation.scss

```diff
@@ -26,6 +26,7 @@
     font-style: normal;
     text-overflow: ellipsis;
     white-space: nowrap;
+    vertical-align: bottom;
```

Comment:
please check if 
` vertical-align: baseline;`
will display this correctly. If so, remove this line as 'baseline' is a default option.

My suggestion is to remove this property because modifying this property causes a lot of o problems in past in ActivitiesBox component.

Reply:
`DeleteForm` component was unused. It was used before when we had an old flow for removing an account. Now we send an email so form and this component is unnecessary.

Reply:
This was my first choice: `vertical-align: baseline`. But it didn't solve the problem. I tried to remove `<em>` tag add `white-space` property etc... But only `vertical-align: bottom` solved the problem.